test/message/test_runner_output.out

```diff
@@ -0,0 +1,327 @@
+      ...
+    1..4
+ok 23 - level 0a
+  ---
+  duration_ms: *
+  ...
+    not ok 1 - +long running
+      ---
+      duration_ms: *
+      failureType: 'cancelledByParent'
+      error: "'test did not finish before its parent and was cancelled'"
+      code: ERR_TEST_FAILURE
+      ...
+        ok 1 - ++short running
+          ---
+          duration_ms: *
+          ...
+        1..1
+    ok 2 - +short running
+      ---
+      duration_ms: *
+      ...
+    1..2
+not ok 24 - top level
+  ---
+  duration_ms: *
+  failureType: 'subtestsFailed'
+  error: "'1 subtest failed'"
+  code: ERR_TEST_FAILURE
+  ...
+ok 25 - invalid subtest - pass but subtest fails
+  ---
+  duration_ms: *
+  ...
+ok 26 - sync skip option # SKIP
+  ---
+  duration_ms: *
+  ...
+ok 27 - sync skip option with message # SKIP this is skipped
+  ---
+  duration_ms: *
+  ...
+not ok 28 - sync skip option is false fail
+  ---
+  duration_ms: *
+  failureType: 'testCodeFailure'
+  error: 'this should be executed'
+  code: ERR_TEST_FAILURE
+  stack: |-
+    *
+    *
+    *
+    *
+    *
+    *
+    *
+  ...
+ok 29 - noop
+  ---
+  duration_ms: *
+  ...
+ok 30 - functionOnly
+  ---
+  duration_ms: *
+  ...
+ok 31 - <anonymous>
+  ---
+  duration_ms: *
+  ...
+ok 32 - test with only a name provided
+  ---
+  duration_ms: *
+  ...
+ok 33 - noop
+  ---
+  duration_ms: *
+  ...
+ok 34 - noop # SKIP
+  ---
+  duration_ms: *
+  ...
+ok 35 - test with a name and options provided # SKIP
+  ---
+  duration_ms: *
+  ...
+ok 36 - functionAndOptions # SKIP
+  ---
+  duration_ms: *
+  ...
+not ok 37 - invalid subtest fail
+  ---
+  duration_ms: *
+  failureType: 'parentAlreadyFinished'
+  error: "'test could not be started because its parent finished'"
+  code: ERR_TEST_FAILURE
+  stack: |-
+    Immediate._onImmediate (*)
+  ...
+1..37
+# Warning: Test "unhandled rejection - passes but warns" generated asynchronous activity after the test ended. This activity created the error "Error: rejected from unhandled rejection fail" and would have caused the test to fail, but instead triggered an unhandledRejection event.
+# Warning: Test "async unhandled rejection - passes but warns" generated asynchronous activity after the test ended. This activity created the error "Error: rejected from async unhandled rejection fail" and would have caused the test to fail, but instead triggered an unhandledRejection event.
+# Warning: Test "immediate throw - passes but warns" generated asynchronous activity after the test ended. This activity created the error "Error: thrown from immediate throw fail" and would have caused the test to fail, but instead triggered an uncaughtException event.
+# Warning: Test "immediate reject - passes but warns" generated asynchronous activity after the test ended. This activity created the error "Error: rejected from immediate reject fail" and would have caused the test to fail, but instead triggered an unhandledRejection event.
+# tests 37
+# pass 11
+# fail 16
```

Comment:
It looks like the skipped tests are counted as fail?

Reply:
I looked at the output and counted 14x `not ok` I was wondering where the 16 came from and skimming the code I thought it was maybe due to the skip calls?!